ppr-ui/src/components/parties/composables/useDebtor.ts

```diff
@@ -92,7 +92,7 @@ export const useDebtor = (props, context) => {
         parseInt(localState.day)
       )
       if (dateOfBirth instanceof Date && !isNaN(dateOfBirth.valueOf())) {
-        localState.currentDebtor.birthDate = dateOfBirth.toUTCString()
+        localState.currentDebtor.birthDate = dateOfBirth.toISOString()
```

Comment:
Will this work? The API needs the long timestamp ISO format with the timezone. You can append "T00:00:00-08:00".